app/services/barbeque/message_enqueuing_service.rb

```diff
@@ -15,11 +15,13 @@ def self.sqs_client
   # @param [String] job
   # @param [Object] message
   # @param [String] queue
-  def initialize(application:, job:, message:, queue: nil)
+  # @param [Integer, nil] delay_seconds
+  def initialize(application:, job:, message:, queue: nil, delay_seconds:)
```

Comment:
Is it ok that `delay_seconds` is an optional argument like `delay_seconds: nil`.